### PR TITLE
build: write a general overview explaining cryptocurrencies

### LINKED ISSUE #491: [Entry Request] General: Cryptocurrency
### Reason/inspiration (optional)

What is cryptocurrency? 
Pretend I'm 5 years old, and explain what Bitcoin is? Ethereum? Dogecoin? DeFi?
Share wisdom on the future of crypto and how it will shape the role of money in society. 

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by 0xTomas**
### Description

Closes #491. This PR provides an overview of cryptocurrencies. It outlines:
- What is cryptocurrency?
- Key Concepts
  - Blockchain
  - Types
  - Purchasing cryptocurrencies
- Advantages and disadvantages

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by ethan-master-coding** [comment]
Make sure to link the issue. Type `Closes #491` in the body of the PR

**Comment by 0xTomas** [comment]
> Make sure to link the issue. Type `Closes #491` in the body of the PR

Thanks, updated.

**Comment by 0xTomas** [comment]
@KTom101 Please review and let me know if you'd like any changes to be made.

**Review by KTom101 — CHANGES_REQUESTED** [review]

**Comment by KTom101** [comment]
@0xTomas Thanks for your entry! I've added my review of this. Please let us know when you're done making changes. Also, you can remove this from draft status when you're ready.

**Comment by 0xTomas** [comment]
@KTom101 Thanks for the comments, updated and ready for review.

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (KTom101) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:28]:
```suggestion
Ethereum, the second most valuable cryptocurrency, features a smart contract platform that allows for numerous protocols including borrowing and lending services. There are also “stable coin” cryptocurrencies, which attempt to offer price stability and are backed by a reserve asset such as US dollars. There are also “altcoins”, some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the Doge meme created from a viral photo of a Shiba Inu.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (KTom101) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:14]:
```suggestion
A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world. Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (KTom101) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:26]:
```suggestion
Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalization. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (KTom101) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:49]:
```suggestion
- _Criminal activity_: Cryptocurrencies have the potential to be used by criminals, although the same is also true for physical cash. In the early days of Bitcoin, there was an online marketplace on the dark web called Silk Road which allowed users to purchase illegal drugs and conduct other nefarious activities.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (KTom101) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:53]:
```suggestion
- _Energy usage_: The mining of popular cryptocurrencies requires substantial energy costs, in some cases as much as entire countries. While that may be true, cryptocurrency supporters argue that the existing financial structure uses far more energy than the power required for mining.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (KTom101) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:3]:
```suggestion
Description: 'A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world.'
```

**Review by KTom101 — COMMENTED** [review]
@0xTomas Noticed a few more things that need changing. After, this should be good for the next review. Thanks!

**Comment by 0xTomas** [comment]
Thanks @KTom101, should be all good to go now.

**Review by KTom101 — APPROVED** [review]
@0xTomas Looks ready for a second review. Thanks!

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hey @0xTomas 👋🏻 Thank you very much for this contribution! I went ahead and gave this PR a review and left some comments/suggestions for you to look at. 😄

**Comment by 0xTomas** [comment]
Thanks for the review @Dusch4593. I have updated the paragraphing of the advantages and disadvantages using headings rather than as a list, let me know your thoughts.

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (Dusch4593) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:23]:
```suggestion
### Mining

Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (Dusch4593) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:21]:
```suggestion
A blockchain is a set of interconnected “blocks” that form an online ledger that is distributed across computers around the world. These linked blocks contain recorded transactions that are verified by nodes in the network. Nodes are the network computers running the software client that locally stores the blockchain data. 

The data is updated as transactions are stored in each block and every copy of the blockchain is updated simultaneously, keeping all the records identical and accurate. As the network relies on consensus mechanisms to verify transactions, it is almost impossible to forge a fraudulent transaction.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hey @0xTomas 👋🏻 Thanks for pushing those changes! After looking it over again, I left a couple more comments/suggestions and after we resolve those those, we'll be good for my review. 😄

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (0xTomas) [reply content/general/concepts/cryptocurrency/cryptocurrency.md:73]: Great point, I will update it to refer to the Makarov and  Schoar (2021) paper you mentioned.

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (0xTomas) [reply content/general/concepts/cryptocurrency/cryptocurrency.md:77]: Interesting point. I have found that in the early days, when people held hundreds of coins, they often referred to them as countable "bitcoins". Nowadays, with it having become much more valuable and more like 'money', there seems to be a preference to use the singular 'Bitcoin'. I will update it to the more contemporary usage.

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:27]:
```suggestion
Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining. Nowadays, the computing power necessary is only achieved through a powerful network of computers known as a mining pool.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:21]:
```suggestion
A blockchain is essentially a set of interconnected "blocks" that form an online ledger that is distributed across computers around the world. These linked blocks contain recorded transactions that are verified by nodes in the network. Nodes are the network computers running the software client that locally stores the blockchain data.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:35]:
```suggestion
There are also **altcoins**,some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the "Doge meme" created from a viral photo of a Shiba Inu.
```

I think we include inline meme names using double quotes

Comment (yangc95) [reply content/general/concepts/cryptocurrency/cryptocurrency.md:35]: @KTom101 @Dusch4593 do we know if it's okay to bold words we want to bring attention to 💭

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:37]:
```suggestion
### Purchasing Cryptocurrencies
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:41]:
```suggestion
## Advantages and Disadvantages
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:43]:
```suggestion
Cryptocurrencies were originally designed with the intent of revolutionizing existing financial infrastructure through greater efficiency and transparency combined with lower costs. There exists, however, a notable divide between the theoretical ideal of a transparent and decentralized system and its practical implementation in the real world.
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:45]:
```suggestion
Outlined below are the primary advantages and disadvantages of cryptocurrencies:
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:83]:
```suggestion
#### Price Volatility
```

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (yangc95) [inline content/general/concepts/cryptocurrency/cryptocurrency.md:87]:
```suggestion
#### Energy Usage
```

**Review by yangc95 — COMMENTED** [review]
@0xTomas had a few comments!

File: content/general/concepts/cryptocurrency/cryptocurrency.md
Comment (0xTomas) [reply content/general/concepts/cryptocurrency/cryptocurrency.md:35]: Great, thanks for the feedback @yangc95. I'll wait until this is clarified then do all the changes at once.

Comment (Dusch4593) [reply content/general/concepts/cryptocurrency/cryptocurrency.md:35]: @yangc95 @0xTomas Per our [Markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#bold), bold should be used for new terms, keywords, or concepts (but not for general emphasis).

Comment (0xTomas) [reply content/general/concepts/cryptocurrency/cryptocurrency.md]: Thanks @Dusch4593, I have updated these changes.

**Review by Dusch4593 — APPROVED** [review]
@0xTomas LGTM! 👍🏻 Thank you so much for pushing those suggested changes. 😄

Before we merge, I wanted to fill you in on some small edits I made here in 94e03a6b1f03bde50c977170fb547f2a7f4232e4:

- I replaced all smart quotes (curly or sloped - `“`) with straight-quotes (`"`) to keep consistent with other existing Docs entries.
- Light word-smithing.
- **Bolded** the first mention of the concept "cryptocurrency". Can we actually continue doing this for concept/term entries going forward?

@KTom101 @yangc95 Any left to add or check before we merge?

**Comment by 0xTomas** [comment]
> @0xTomas LGTM! 👍🏻 Thank you so much for pushing those suggested changes. 😄
> 
> Before we merge, I wanted to fill you in on some small edits I made here in [94e03a6](https://github.com/Codecademy/docs/commit/94e03a6b1f03bde50c977170fb547f2a7f4232e4):
> 
> * I replaced all smart quotes (curly or sloped - `“`) with straight-quotes (`"`) to keep consistent with other existing Docs entries.
> * Light word-smithing.
> * **Bolded** the first mention of the concept "cryptocurrency." Can we actually continue doing this for concept/term entries going forward?
> 
> @KTom101 @yangc95 Any left to add or check before we merge?

Beauty, thanks for the update.

**Comment by Dusch4593** [comment]
> Beauty, thanks for the update.

@0xTomas Anytime 😄 If you want to learn more about our markdown style preferences (which recently updated back in February), you can do so [here](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/).

**Comment by KTom101** [comment]
@Dusch4593 I think that's all. LGTM!

**Comment by Dusch4593** [comment]
@0xTomas Congrats on your first Docs contribution! It's now live: https://www.codecademy.com/resources/docs/general/cryptocurrency

I noticed, however, that you weren't listed as a contributor on the right-hand side:

Screen Shot 2022-06-04 at 6 48 35 PM

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username "rzvl" (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. 😄

**Comment by 0xTomas** [comment]
> @0xTomas Congrats on your first Docs contribution! It's now live: https://www.codecademy.com/resources/docs/general/cryptocurrency
> 
> I noticed, however, that you weren't listed as a contributor on the right-hand side:
> 
> Screen Shot 2022-06-04 at 6 48 35 PM
> 
> If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:
> 
> 1. Go to your Codecademy dashboard.
> 2. Click on your profile image in the top-right and then choose "Profile".
> 3. Then click "Edit Profile".
> 4. In the "GitHub Username" field, add your username "rzvl" (without the @).
> 5. Click "Save Changes"
> 
> Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. 😄

As far as I can tell I have added my GitHub username to my Codecademy profile:

image

Does this look right?

**Comment by Dusch4593** [comment]
> Does this look right?

@0xTomas Hmmm no luck on my end. Have you tried logging out of your Codcademy profile and re-logging through GitHub?

Screen Shot 2022-06-04 at 10 55 57 PM

**Comment by 0xTomas** [comment]
I have logged in and authorised through GitHub, and it looks like it is connected now:

image

**Comment by Dusch4593** [comment]
It looks like it did! However, is the profile that's now appearing on the entry yours or somebody else's?

Screen Shot 2022-06-04 at 11 14 08 PM

**Comment by 0xTomas** [comment]
Yes, although I just updated my Codecademy username to match my Github username, and also remove my first and last name, so hopefully that updates at some point.

**Pull request merged**
